tests/imap/data.py

```diff
@@ -9,6 +9,8 @@
 from flanker import mime
 from hypothesis import strategies as s
 
+MAX_INT_VALUE = (1 << 32) - 1
```

Comment:
This change id unrelated but removes test flakiness that I saw when opening PR. Sometimes we would get test data that would be outside mysql int range.